GSA/digitalgov.gov
# POAM - April - Update dependencies

## Summary

Part of POAM for April. Updating dependencies.

### Preview

[Link to Preview]()

### Solution

1. Update dependencies
2. Remove unused `gulp-size`
3. Remove unused `gulp-dart-scss`
4. Add override for `glob-parent`

### Dependencies

| Depdency                    | Old    | New    |
| :-------------------------- | :----- | :----- |
| @prantlf/jsonlint           | -      | 14.0.3 |
| del                         | 3.0.0  | 6.1.1  |
| glob-parent _via overrides_ | -      | 6.0.2  |
| gulp-dart-scss              | 1.1.0  | -      |
| gulp-size                   | 3.0.0  | -      |
| jsonlint                    | 1.6.3  | -      |
| markdownlint-cli            | 0.32.2 | 0.33.0 |
| postcss                     | 8.4.21 | 8.4.23 |
| prettier                    | 2.8.7  | 2.8.8  |
| sass                        | 1.60.0 | 1.62.1 |
| sass-embedded               | 1.60.0 | 1.62.0 |

### Major changes

Previous jsonlint package is no longer maintained. I've replaced it with a more up-to-date package in `@prantlf/jsonlint`. JSON linting should continue working as expected.


### How To Test

1. Run `npm install`
2. There should be 0 vulnerabilities
3. Lint JSON task should run without issues
    1. Run `hugo` in terminal
    2. Run `npm run lint:json`

---

### Dev Checklist
